www/src/models/user.ts

```diff
@@ -0,0 +1,58 @@
+import m from "mithril";
+
+export interface User {
+    email: string;
+    userName: string;
+    password: string;
+    fullName: string;
+}
+
+const UserModel = {
+    current: {} as User,
+    load(userName: string) {
+        return m.request<User>({
+            method: "GET",
+            url: "/user/" + userName,
+
+        })
+            .then(result => {
+                UserModel.current = result;
+            })
+    },
+
+    save() {
+        return m.request ({
+            method: "PATCH",
+            url: "/user/" + userName,
+            data: UserModel.current,
+        })
+    },
+    // TODO: make shure request data is sent as JSON, applies for all commands.
```

Comment:
I _think_ Mithril will send this as JSON:

![2018-11-25-115256_845x235_scrot](https://user-images.githubusercontent.com/4541181/48978232-b3696180-f0a8-11e8-843f-fbef4f6d737d.png)

So it defaults to `JSON.stringify` :)